share/completions/konsole.fish

```diff
@@ -1,35 +1,30 @@
-complete -c konsole -l author -d 'Show author information and exit'
-complete -c konsole -l license -d 'Show license information and exit'
-complete -c konsole -l name -d 'Window class'
-complete -c konsole -l ls -d 'Login shell'
-complete -c konsole -s T -d 'Window name'
-complete -c konsole -l tn -d 'Terminal type'
-complete -c konsole -l noclose -d 'Do not close when command exists'
-complete -c konsole -l nohist -d 'Do not save history'
-complete -c konsole -l nomenubar -d 'Hide menu bar'
-complete -c konsole -l notabbar -d 'Hide tab bar'
-complete -c konsole -l notoolbar -d 'Hide tab bar'
-complete -c konsole -l noframe -d 'Hide frame bar'
-complete -c konsole -l noscrollbar -d 'Hide scrollbar'
-complete -c konsole -l noxft -d 'Do not use atni-aliasing'
-complete -c konsole -l noresize -d 'Do not allow window to be resized'
-complete -c konsole -l vt_sz -d 'Window size'
-complete -c konsole -l type -d 'Session type'
-complete -c konsole -l types -d 'List session types'
-complete -c konsole -l keytab -d 'Keytab name'
-complete -c konsole -l keytabs -d 'List keytabs'
-complete -c konsole -l profile -d Profile
-complete -c konsole -l profiles -d 'List profiles'
-complete -c konsole -l schema -d Schema
-complete -c konsole -l schemas -d 'List schemas'
-complete -c konsole -l script -d 'Enable extended DCOP Qt functions'
-complete -c konsole -l workdir -d 'Working directory'
-complete -c konsole -s e -d 'Command to execute'
+complete -c konsole -l version -xs v -d 'Show version and exit'
+complete -c konsole -l author -d 'Show author info and exit'
+complete -c konsole -l license -d 'Show license info and exit'
+complete -c konsole -l desktopfile -d 'Base file name of desktop entry'
+complete -c konsole -l profile -d 'Name of profile to use' -xa "(konsole --list-profiles)"
+complete -c konsole -l layout -r -d 'json layoutfile to be loaded'
+complete -c konsole -l builtin-profile -d 'Use the built-in profile'
+complete -c konsole -l workdir -d 'Set initial working dir' -xa "(__fish_complete_directories)"
+complete -c konsole -l hold -l noclose -d 'Do not close when command exists'
+complete -c konsole -l new-tab -d 'Create a new tab'
+complete -c konsole -l tabs-from-file -r -d 'Create tabs from tabs config'
+complete -c konsole -l background-mode -d 'Start Konsole in the background'
+complete -c konsole -l separate -l nofork -d 'Run in a separate process'
+complete -c konsole -l show-menubar -d 'Show the menubar'
+complete -c konsole -l hide-menubar -d 'Hide the menubar'
+complete -c konsole -l show-tabbar -d 'Show the tabbar'
+complete -c konsole -l hide-tabbar -d 'Hide the tabbar'
+complete -c konsole -l fullscreen -d 'Start Konsole in fullscreen mode'
+complete -c konsole -l notransparency -d 'Disable transparent backgrounds'
+complete -c konsole -l list-profiles -d 'List the available profiles'
+complete -c konsole -l list-profile-properties -d 'List all the profile properties'
+complete -c konsole -xs p -d '<P=V> Change profile property'
```

Comment:
It would be nice to dynamically complete `konsole -p' by providing the options to it, perhaps `-xa "(konsole --list-profile-properties)="` (would need to be tested)

Reply:
The output of --list-profile-properties is in the form of `Property : Type`, such as `Path : QString`, so I'm thinking of using: konsole --list-profile-properties | sed "s/ : /=/g" to format the output to `Property=Type`

Reply:
Well, it expects `key=<arbitrary value>` so putting it in `key=<type>` doesn't help. I would just use `string replace -fr '(.*?) .*' '$1='` to put it in the form and have the user provide the arbitrary value. Can you try that, does it work?